examples/vision/siamese.py

```diff
@@ -0,0 +1,290 @@
+
+"""
+# Siamese network with a contrastive loss
+
+Author: [Mehdi](https://github.com/s-mrb)<br>
+Date created: 2021/05/06<br>
+Last modified: 2020/05/06<br>
+Description: Similarity learning using siamese network with contrastive loss
+"""
+
+"""
+## Introduction
+
+[Siamese Network](https://en.wikipedia.org/wiki/Siamese_neural_network)
```

Comment:
If I separate [Siamese Network] and its associated link in parenthesis by newline then it do not appear as hyperlink in Colab.

Reply:
The suggestion I had earlier was this:

```
The [Siamese network](https://papers.nips.cc/paper/1993/file/288cc0ff022877bd3df94bc9360b9c5d-Paper.pdf)
```

which is in one line.

Anyway, this is just a suggestion.